### PR TITLE
Proposal: MOBILE Network Onboarding Fee

### PR DESCRIPTION
**Pull request by heatedlime**
This HIP proposes implementing a $40 onboard fee paid in Data Credits (DC) for each Hotspot onboarded to the Mobile Network.

### CODE REVIEW

**Comment by KeithRettig** [comment]
I will stop editing now.  All done.

**Comment by KeithRettig** [comment]
Final edit.  Requested changes made.

File: 0000-designate-mobile-network-onboarding-fees-and-device-definition.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+## Alternatives
+One alternative is to do nothing, and keep Onboarding Fees as $0$ (zero). However, this would impair the daily HNT emissions to the MOBILE subDAO treasury since the $A$ score would remain $1$ (one). 
+
+Another alternative is to change the way the $A$ score is calculated to benefit the HNT emissions of the MOBILE Network; however, this would require a vote with veHNT instead of veMOBILE. 
+
+A complementary HIP is to require that for each MOBILE Hotspot that was previously onboarded at 0 USD, there is to be 40 USD worth of DCs burned to the MOBILE subDAO.  Such a HIP should not require any specific payer for the DCs; it would be the responsibility of the MOBILE subDAO to find the payer(s) and to ensure the payment is settled.  Given there is no mechanism currently for this be done, such a feature can not be included in this HIP.
+
+## Deployment Impact
+After the passing of this HIP, the Helium Foundation will need to modify the chain variable that defines the onboarding fee for the MOBILE subDAO.
```
Comment (abhay) [inline 0000-designate-mobile-network-onboarding-fees-and-device-definition.md:40]:
```suggestion
After the passing of this HIP, the Helium multisig will need to modify the variable that defines the onboarding fee for the MOBILE subDAO.
```

File: 0000-designate-mobile-network-onboarding-fees-and-device-definition.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+- Category: Technical & Economic
+- Original HIP PR: #
+- Tracking Issue: #
+- Voting Requirements: veMOBILE
+
+## Summary
+This HIP proposes implementing a $40 onboard fee paid in Data Credits (DC) for each Hotspot onboarded to the MOBILE Network.
+
+## Motivation
+Current MOBILE Hotspots, such as FreedomFi and Bobcat 500, contain both an IOT Hotspot and a MOBILE Hotspot. As such, HIP-53 specifies that MOBILE Hotspots should burn a 40 USD Onboarding Fee to be onboarded to the network. All onboarded MOBILE Hotspots were onboarded to the IOT network with the correct Onboarding Fee of 40 USD, but were onboarded to the MOBILE network with an Onboarding Fee of 0 USD. As a result, 0 USD in Onboarding Fees have been burned towards the MOBILE subDAO's $A$ Factor, which negatively affects the $A$ Score established in HIP 51 for the MOBILE subDAO. The current $A$ Score within the DAO Utility Score is as follows:
```
Comment (abhay) [inline 0000-designate-mobile-network-onboarding-fees-and-device-definition.md:13]:
```suggestion
Current MOBILE Hotspots, such as FreedomFi and Bobcat 500, contain both IOT and a MOBILE network capabilities. As such, HIP-53 specifies that MOBILE Hotspots should burn a 40 USD Onboarding Fee to be onboarded to the network. All onboarded MOBILE Hotspots were onboarded to the IOT network with the correct Onboarding Fee of 40 USD, but were onboarded to the MOBILE network with an Onboarding Fee of 0 USD. As a result, 0 USD in Onboarding Fees have been burned towards the MOBILE subDAO's $A$ Factor, which negatively affects the $A$ Score established in HIP 51 for the MOBILE subDAO. The current $A$ Score within the DAO Utility Score is as follows:
```

**Review by abhay — COMMENTED** [review]
minor comments.

**Pull request merged**